Process.c

```diff
@@ -821,34 +836,13 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
 
       xSnprintf(buffer, n, "%4ld ", this->nlwp);
       break;
-   case PERCENT_CPU:
+   case PERCENT_CPU: Process_printPercentage(this->percent_cpu, buffer, n, &attr); break;
    case PERCENT_NORM_CPU: {
-      float cpuPercentage = this->percent_cpu;
-      if (field == PERCENT_NORM_CPU) {
-         cpuPercentage /= this->processList->activeCPUs;
-      }
-      if (cpuPercentage > 999.9F) {
-         xSnprintf(buffer, n, "%4u ", (unsigned int)cpuPercentage);
-      } else if (cpuPercentage > 99.9F) {
-         xSnprintf(buffer, n, "%3u. ", (unsigned int)cpuPercentage);
-      } else {
-         if (cpuPercentage < 0.05F)
-            attr = CRT_colors[PROCESS_SHADOW];
-
-         xSnprintf(buffer, n, "%4.1f ", cpuPercentage);
-      }
+      float cpuPercentage = this->percent_cpu / this->processList->activeCPUs;
+      Process_printPercentage(cpuPercentage, buffer, n, &attr);
       break;
    }
-   case PERCENT_MEM:
-      if (this->percent_mem > 99.9F) {
-         xSnprintf(buffer, n, "100. ");
-      } else {
-         if (this->percent_mem < 0.05F)
-            attr = CRT_colors[PROCESS_SHADOW];
-
-         xSnprintf(buffer, n, "%4.1f ", this->percent_mem);
-      }
-      break;
+   case PERCENT_MEM: Process_printPercentage(this->percent_mem, buffer, n, &attr); break;
```

Comment:
Line wrapping and indent (let's avoid formatting case statements as one line unless all other case statements are done so):
```suggestion
   case PERCENT_MEM:
      Process_printPercentage(this->percent_mem, buffer, n, &attr);
      break;
```

Reply:
Same here. Maybe another time in another PR, but currently I'd prefer to avoid that noise.